AntiCovid/app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,74 @@
+<resources>
+    <string name="app_name">反Covid</string>
+    <string name="profile">轮廓</string>
+    <string name="add_test_report">添加测试报告t</string>
+    <string name="my_vaccine_information">我的疫苗信息</string>
+    <string name="valid_qr_code">有效的QR码</string>
+    <string name="my_previous_test_reports">我以前的测试报告</string>
+    <!-- Strings related to login -->
+    <string name="prompt_email">电子邮件</string>
+    <string name="prompt_password">密码</string>
+    <string name="action_back">后退</string>
+    <string name="action_sign_in">注册</string>
+    <string name="action_sign_in_short">登入</string>
+    <string name="welcome">"欢迎!"</string>
+    <string name="invalid_username">无效的用户名</string>
+    <string name="invalid_password">密码必须大于5个字符</string>
+    <string name="login_failed">登录失败</string>
+    <string name="create_acc">创建帐号</string>
+    <string name="address">地址</string>
+    <string name="social_sec_id">身分证字号</string>
+    <string name="phone_number">电话号码</string>
+    <string name="name">名称</string>
+    <string name="surname">姓</string>
+    <string name="register_title">创建帐号</string>
+    <string name="no_vaccine_info">我们没有有关您的疫苗接种的信息。</string>
+    <string name="recently_vaccined_question">您最近接种过疫苗吗？</string>
+    <string name="insert_id_for_vaccine_qr_code">如果是这样，请添加您的疫苗ID以生成永久的QR码</string>
+    <string name="qr_code_created_after_second_dose">请记住，当您获得第二份文件时，将生成永久QR码</string>
+    <string name="add_vaccine_id">添加疫苗ID</string>
+    <string name="add_vaccine">添加疫苗</string>
+    <string name="manufacturer">制造商</string>
+    <string name="first_dose_date">首次服药日期</string>
+    <string name="second_dose_date">第二次服药日期</string>
+    <string name="institution">机构</string>
+    <string name="accept">接受</string>
+    <string name="accept_terms_of_use">接受使用条款</string>
```

Comment:
I dont know how much work it would be, but there is also a Terms of Use file saved in the assets folder. Could you also add that translation and read the correct file to show the Terms of Use ?

Reply:
Thanks